src/neptune/api/field_visitor.py

```diff
@@ -39,12 +39,16 @@
     StringField,
     StringSeriesField,
     StringSetField,
+    UnknownField,
 )
 from neptune.exceptions import MetadataInconsistency
 
 
 class FieldToValueVisitor(FieldVisitor[Any]):
```

Comment:
`FieldVisitor` should be updated as well

Reply:
We have a `FieldVisitor.visit_unknown()`